server/src/main/java/com/distkv/server/metaserver/server/bean/GetGlobalViewResponse.java

```diff
@@ -0,0 +1,58 @@
+package com.distkv.server.metaserver.server.bean;
+
+import com.distkv.server.view.NodeTable;
+
+import java.io.Serializable;
+import java.util.concurrent.ConcurrentHashMap;
+
+public class GetGlobalViewResponse implements Serializable {
+
+  private static final long serialVersionUID = -7220117786727166673L;
+
+  private ConcurrentHashMap<String, NodeTable> globalView;
+
+  /**
+   * Whether the request is executed successfully.
+   */
+  private boolean success;
+  /**
+   * Redirect to another peer id. That means the current peer is not leader,
+   * and it redirect to the new leader.
+   */
+  private String redirect;
+
+  private String errorMessage;
+
+  public boolean isSuccess() {
+    return success;
+  }
+
+  public void setSuccess(boolean success) {
+    this.success = success;
+  }
+
+  // TODO(qwang): Use this method on client side.
+  public String getRedirect() {
```

Comment:
A BaseClass ?

Reply:
I don't understand what you mean. Can you explain it in more detail?